orangecontrib/text/string_transformation.py

```diff
@@ -0,0 +1,72 @@
+import collections
+import re
+from bs4 import BeautifulSoup
+from sklearn.feature_extraction.text import strip_accents_unicode
+
+__all__ = ["TextStringTransformer", "HtmlStringTransformer",
+           "WhitespaceStringTransformer"]
+
+
+class BaseStringTransformer:
+    def __call__(self, data):
+        """ Transforms strings in given object.
+        :param data: string or string collection.
+        :type data: str or Iterable
```

Comment:
Regardless of what it may appear from existing code, we prefer [Napoleon-style docstrings](http://sphinxcontrib-napoleon.readthedocs.org/en/latest/index.html) in new code.